packages/phone-number-privacy/package.json

```diff
@@ -29,7 +29,8 @@
   },
   "devDependencies": {
     "@types/uuid": "^7.0.3",
-    "firebase-functions-test": "^0.2.0"
+    "firebase-functions-test": "^0.2.0",
+    "query-string": "^6.12.1"
```

Comment:
Where do we use this?

Reply:
Ah I added this because I was writing some e2e tests.. will exclude from this PR